.github/workflows/build_wheels.yml

```diff
@@ -144,15 +144,15 @@ jobs:
           python -m pip install toml
           python build-scripts/environ-from-pyproject.py >> $GITHUB_ENV
 
+      - name: Add /usr/local/lib to dynamic linker path
+        run: echo "/usr/local/lib" | sudo tee /etc/ld.so.conf.d/local-lib.conf
+
```

Comment:
Do you know why these linker changes were necessary?

Reply:
Well, tbh, this one isn't strictly needed. Really the change is to call `ldconfig` after installing libraries, so that they were properly found in the library cache.

I made this change more for consistency. On one container image, `/usr/local/lib` is on the library path, while on this one, it isn't. So this puts it on there directly, which I admit is effectively the same as `LD_LIBRARY_PATH`, except it's more 'permanent'.

Reply:
PS, you know you can just cherry-pick the commit from here?

Reply:
I didn't realize you had the PR already split it into two commits. Sorry about that.